app/src/main/java/com/android/unio/ui/event/EventDetails.kt

```diff
@@ -433,13 +443,26 @@ fun EventDetailsPicturesTab(event: Event, context: Context) {
                     context.getString(R.string.event_details_user_picture_content_description),
                 filterQuality = FilterQuality.Medium,
                 placeholderResourceId = 0,
-                contentScale = ContentScale.Crop,
+                contentScale = ContentScale.Fit,
                 modifier =
                     Modifier.padding(3.dp)
                         .clip(RoundedCornerShape(10))
-                        .testTag(EventDetailsTestTags.USER_EVENT_PICTURE + item.uid))
+                        .testTag(EventDetailsTestTags.USER_EVENT_PICTURE + item.uid)
+                        .clickable {
+                          scope.launch { pagerState.scrollToPage(index) }
+                          showFullScreen = true
+                        })
           }
         }
+    if (showFullScreen) {
+      PictureOverlay(
+          onDismiss = {
+            selectedPictureUri = Uri.EMPTY
+            showFullScreen = false
+          },
+          pagerState,
+          eventPictures)
+    }
```

Comment:
Is this supposed to be from the click on image feature addition? Why does it show up like an addition? Don't tell me you accidentally rebased again 🤣. Anyway just make sure it's on par with the main branch, so that it doesn't crush anything.